include/dfrn.php

```diff
@@ -16,6 +26,10 @@
  */
 class dfrn {
```

Comment:
This class existed before @fabrixxm said this :-) I just added some more functions to this class.